neurodocker/templates/neurodebian.yaml

```diff
@@ -2,20 +2,20 @@
 # Instructions to add NeuroDebian repositories.
 
 name: neurodebian
-url: http://neuro.debian.net
+url: https://neuro.debian.net
 binaries:
     urls:
-        australia: http://neuro.debian.net/lists/{{ self.os_codename }}.au.{{ self.full_or_libre }}
-        china-tsinghua: http://neuro.debian.net/lists/{{ self.os_codename }}.cn-bj1.{{ self.full_or_libre }}
-        china-scitech: http://neuro.debian.net/lists/{{ self.os_codename }}.cn-bj2.{{ self.full_or_libre }}
-        china-zhejiang: http://neuro.debian.net/lists/{{ self.os_codename }}.cn-zj.{{ self.full_or_libre }}
-        germany-munich: http://neuro.debian.net/lists/{{ self.os_codename }}.de-m.{{ self.full_or_libre }}
-        germany-magdeburg: http://neuro.debian.net/lists/{{ self.os_codename }}.de-md.{{ self.full_or_libre }}
-        greece: http://neuro.debian.net/lists/{{ self.os_codename }}.gr.{{ self.full_or_libre }}
-        japan: http://neuro.debian.net/lists/{{ self.os_codename }}.jp.{{ self.full_or_libre }}
-        usa-ca: http://neuro.debian.net/lists/{{ self.os_codename }}.us-ca.{{ self.full_or_libre }}
-        usa-nh: http://neuro.debian.net/lists/{{ self.os_codename }}.us-nh.{{ self.full_or_libre }}
-        usa-tn: http://neuro.debian.net/lists/{{ self.os_codename }}.us-tn.{{ self.full_or_libre }}
+        australia: https://neuro.debian.net/lists/{{ self.os_codename }}.au.{{ self.full_or_libre }}
+        china-tsinghua: https://neuro.debian.net/lists/{{ self.os_codename }}.cn-bj1.{{ self.full_or_libre }}
+        china-scitech: https://neuro.debian.net/lists/{{ self.os_codename }}.cn-bj2.{{ self.full_or_libre }}
+        china-zhejiang: https://neuro.debian.net/lists/{{ self.os_codename }}.cn-zj.{{ self.full_or_libre }}
+        germany-munich: https://neuro.debian.net/lists/{{ self.os_codename }}.de-m.{{ self.full_or_libre }}
+        germany-magdeburg: https://neuro.debian.net/lists/{{ self.os_codename }}.de-md.{{ self.full_or_libre }}
+        greece: https://neuro.debian.net/lists/{{ self.os_codename }}.gr.{{ self.full_or_libre }}
+        japan: https://neuro.debian.net/lists/{{ self.os_codename }}.jp.{{ self.full_or_libre }}
+        usa-ca: https://neuro.debian.net/lists/{{ self.os_codename }}.us-ca.{{ self.full_or_libre }}
+        usa-nh: https://neuro.debian.net/lists/{{ self.os_codename }}.us-nh.{{ self.full_or_libre }}
+        usa-tn: https://neuro.debian.net/lists/{{ self.os_codename }}.us-tn.{{ self.full_or_libre }}
```

Comment:
have you checked all those urls to provide https with correct certificates etc?

in principle -- yes, nice to use https but might be "more fragile" I guess and again -- not guaranteed they all have it

Reply:
@yarikoptic seems to work:


```console
[deco]/tmp ❱ wget https://neuro.debian.net/lists/bookworm.de-fzj.libre
--2024-04-12 15:19:16--  https://neuro.debian.net/lists/bookworm.de-fzj.libre
Resolving neuro.debian.net... 129.170.233.11
Connecting to neuro.debian.net|129.170.233.11|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 210
Saving to: ‘bookworm.de-fzj.libre’

bookworm.de-fzj.libre              100%[================================================================>]     210  --.-KB/s    in 0s

2024-04-12 15:19:16 (124 MB/s) - ‘bookworm.de-fzj.libre’ saved [210/210]
```

Reply:
sorry -- I was "tired" I guess, mixed up assuming it was http vs https for actual mirrors not our website. Yeah -- we should be good. I think it is worth it indeed.

Reply:
but I need to adjust PR title since it is not specific (use https for what???)